web/themes/custom/usagov/templates/views-view-list--benefit-search-form.html.twig

```diff
@@ -43,7 +43,8 @@
 
     <fieldset>
       {% if node.field_benefits_search_title.value | render %}
-        <legend><h2 id="categories-label" class="font-serif-lg">{{ node.field_benefits_search_title.value }}</h2></legend>
+{#        <legend><h2 id="categories-label" class="font-serif-lg">{{ node.field_benefits_search_title.value }}</h2></legend>#}
+        <legend id="categories-label" class="font-serif-lg">{{ node.field_benefits_search_title.value }}</legend>
```

Comment:
There was some question whether putting an h2 inside a legend was actually okay in HTML5.